ci-operator/step-registry/upi/install/aws/cluster/upi-install-aws-cluster-commands.sh

```diff
@@ -72,6 +71,16 @@ export BOOTSTRAP_URI
 # begin bootstrapping
 if openshift-install coreos print-stream-json 2>/tmp/err.txt >coreos.json; then
   RHCOS_AMI="$(jq -r --arg region "$AWS_REGION" '.architectures.x86_64.images.aws.regions[$region].image' coreos.json)"
+  BOOTSTRAP_INSTANCE_TYPE=i3.large
+  MASTER_INSTANCE_TYPE=m5.xlarge
+  WORKER_INSTANCE_TYPE=m5.xlarge
+  if [[ "${CLUSTER_TYPE}" == "aws-arm64" ]]; then
+    RHCOS_AMI="$(jq -r --arg region "$AWS_REGION" '.architectures.aarch64.images.aws.regions[$region].image' coreos.json)"
+    BOOTSTRAP_INSTANCE_TYPE=m6g.large
+    MASTER_INSTANCE_TYPE=m6g.xlarge
+    WORKER_INSTANCE_TYPE=m6g.xlarge
+    CLUSTER_TYPE=aws 
```

Comment:
Rather than have to change the value of `CLUSTER_TYPE` here, I think it makes more sense to hardcode all the later occurences of `CLUSTER_TYPE` to `aws`, because I believe they are all of the form:
`/var/lib/openshift-install/upi/${CLUSTER_TYPE}/cloudformation/01_vpc.yaml"`

AFAIK there is no point in having a variable in this path. It should just be:
`/var/lib/openshift-install/upi/aws/cloudformation/01_vpc.yaml"`

@rna-afk WDYT?

Reply:
It was there in the templates just because multiple cluster types were supported and yeah it's not needed for this PR.